docs/settings/logging-settings.asciidoc

```diff
@@ -4,172 +4,87 @@
 
-| optimize | console | error
+| `logging.appenders[].<appender-name>.policy.type`
+| Specify the triggering policy for when a rollover should occur for the `rolling-file` type appender. 
 
-| telemetry | json-file-appender | all
+| `logging.appenders[].<appender-name>.policy.interval`
+| Specify the time interval for rotating a log file for a `time-interval` type `rolling-file` appender. *Default 24h*
 
-| metrics.ops | console | debug
-|===
+| `logging.appenders[].<appender-name>.policy.size`
+| Specify the size limit at which the policy should trigger a rollover for a `size-limit` type `rolling-file` appender. *Default 100mb*. 
+
+| `logging.appenders[].<appender-name>.policy.interval`
+| Specify the time interval at which the policy should trigger a rollover for a time-interval type `rolling-file` appender. 
+
+| `logging.appenders[].<appender-name>.policy.modulate`
+| Whether the interval should be adjusted to cause the next rollover to occur on the interval boundary. Boolean.
+
+| `logging.appenders[].<appender-name>.strategy.type`
+| Rolling file strategy type. Only `numeric` is currently supported.
+
+| `logging.appenders[].<appender-name>.strategy.pattern`
+| The suffix to append to the file path when rolling. Must include `%i`.
+
+| `logging.appenders[].<appender-name>.strategy.max`
+| The maximum number of files to keep. Optional. Default is `7`.
 
-NOTE: If you modify `root.appenders`, make sure to include `default`.
+| `logging.appenders[].<appender-name>.layout.type`
+| Determines how the log messages are displayed. Options are `pattern` or `json`. Required.
 
-// For more details about logging configuration, refer to the logging system documentation (update to include a link).
+| `logging.appenders[].<appender-name>.layout.highlight`
+| Optional boolean for to highlight log messages in color. Applies to `pattern` layout only.
+
+| `logging.appenders[].<appender-name>.layout.pattern`
+| Optional <<pattern-layout, string pattern>> for placeholders that will be replaced with data from the actual log message. Applicable to pattern type layout only.
+
+| `logging.root.appenders[]`
+| List of specific appenders to apply to `root`. Defaults to `console` with `pattern` layout.
+
+| `logging.root.level`
+| Specify default verbosity for all log messages to fall back to if not specifically configured at the individual logger level. Options are all, fatal, error, warn, info, debug, trace, off. The all and off levels can be used only in configuration and are just handy shortcuts that allow you to log every log record or disable logging entirely or for a specific logger. Default is `info`.
+
+| `logging.loggers[].<logger>.name:`
+| Specific logger instance.
+
+| `logging.loggers[].<logger>.level`
+| Specify verbosity of log messages for <logger> context. Optional and falls back to `root` level if not configured.
```

Comment:
```suggestion
| Specify verbosity of log messages for <logger> context. Optional and falls back to the verbosity of the `root` logger if not configured.
```

I think this should be _root logger_ not _root level_ correct?

Reply:
Reading the other docs, seems like this should be "inherits the verbosity of any ancestor logger, up to the `root` logger"?

Reply:
Good point! I've changed the wording to your recommendation.